src/app/components/modal/index.html

```diff
@@ -150,7 +150,9 @@
   <sky-demo-page-content sectionHeading="Accessibility">
     <p>Follow these guidelines to ensure that people can use modals with assistive technology or keyboards.</p>
     <ul>
-      <li>When a modal opens, focus should be on the first focusable item in the modal. If the modal does not include focusable items, then focus should be on the modal container element.</li>
+      <li>When a modal opens, focus should be on the first focusable item in the modal. If the modal does not include focusable items, then focus should be on the modal container element.
+      <li>If placing focus on the first interactive element causes the content to scroll out of view, add <sky-code>tabindex=-1</sky-code> to a static element at the top of a modal. For example, add <sky-code>tabindex=-1</sky-code> to the title or first paragraph to initially focus the element. For more best practices, see <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal">W3C's WAI-ARIA Authoring Practices for modals</a>.
```

Comment:
Should we have "on" after "initially focus" in the second sentence? Seems like to me, but I'm not sure.

Also, is the link in the last sentence really specific to this bullet? It doesn't seem like it to me, so I'd suggest moving that last line to make it a standalone paragraph after the bullet list.